bridge/client/app/evaluation-board/evaluation-board.component.ts

```diff
@@ -0,0 +1,71 @@
+import {ChangeDetectorRef, Component, OnDestroy, OnInit} from '@angular/core';
+import {map, takeUntil} from "rxjs/operators";
+import {Subject, Subscription} from "rxjs";
+import {ActivatedRoute, Router} from "@angular/router";
+import {Location} from "@angular/common";
+
+import {Root} from "../_models/root";
+
+import {DataService} from "../_services/data.service";
+import {ApiService} from "../_services/api.service";
+import DateUtil from "../_utils/date.utils";
+import {labels, Trace} from "../_models/trace";
+
+@Component({
+  selector: 'app-project-board',
+  templateUrl: './evaluation-board.component.html',
+  styleUrls: ['./evaluation-board.component.scss']
+})
+export class EvaluationBoardComponent implements OnInit, OnDestroy {
+
+  private unsubscribe$ = new Subject();
+
+  public error: string = null;
+  public contextId: string;
+  public root: Root;
+  public evaluations: Trace[];
+
+  constructor(private _changeDetectorRef: ChangeDetectorRef, private router: Router, private location: Location, private route: ActivatedRoute, private dataService: DataService, private apiService: ApiService) { }
+
+  ngOnInit() {
+    this.route.params
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe(params => {
+        if(params["shkeptncontext"]) {
+          this.contextId = params["shkeptncontext"];
+          this.apiService.getTraces(this.contextId)
+            .pipe(
+              map(response => response.body),
+              map(result => result.events||[]),
+              map(traces => traces.map(trace => Trace.fromJSON(trace)).sort((a, b) => new Date(a.time).getTime() - new Date(b.time).getTime()))
+            )
+            .pipe(takeUntil(this.unsubscribe$))
+            .subscribe((traces: Trace[]) => {
+              if(traces.length > 0) {
+                this.root = Root.fromJSON(traces[0]);
+                this.root.traces = traces;
+                this.evaluations = traces.filter(t => t.type == 'sh.keptn.events.evaluation-done' && (!params["eventselector"] || t.id == params["eventselector"] || t.data.stage == params["eventselector"])) ;
+              } else {
+                this.error = "contextError";
+              }
+            }, (err) => {
+              this.error = "error";
```

Comment:
This error messages do not help much about what the problem is.
For example: in the context error you could add what is the given context and in the second one you could add more information about what went wrong.

Reply:
Actually, the contextError says, that there couldn't be found any traces with that `contextId`.

https://github.com/keptn/keptn/blob/ba5bc699920625e73818b36fd3ac653aec186f32/bridge/client/app/evaluation-board/evaluation-board.component.html#L8-L11

And the "error"-case might only be in case of a error in the backend, and for this we have already a separate issue #1987